### PR TITLE
Add a doc on writing a good support request

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: community/good-member/writing-support-requests.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# How to write a good support request
+
+Want help with an Exercism?
+Help boost your chances of getting a good response by learning how to write a good support request and encouraging our community to support you.
+
+Here are some key tips.
+
+## Use codeblocks, not images
```
Comment (ee7) [inline community/good-member/writing-support-requests.md]:
```suggestion
## Use code blocks, not images
```

File: community/good-member/writing-support-requests.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+When you want to share code (either that you've written, or instructions), use a codeblock, not an image.
```
Comment (ee7) [inline community/good-member/writing-support-requests.md]:
```suggestion
When you want to share code (either that you've written, or instructions), use a code block, not an image.
```

File: community/good-member/writing-support-requests.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+````
+
+That will get rendered like this:
+
+```
+for number in range(10):
+  total += number;
+```
+
+You can even get syntax highlighting by adding the language to the codeblock opening:
```
Comment (ee7) [inline community/good-member/writing-support-requests.md]:
```suggestion
You can even get syntax highlighting by adding the language to the code block opening:
```

File: community/good-member/writing-support-requests.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+Help boost your chances of getting a good response by learning how to write a good support request and encouraging our community to support you.
+
+Here are some key tips.
+
+## Use codeblocks, not images
+
+When you want to share code (either that you've written, or instructions), use a codeblock, not an image.
+
+As developers, we're much quicker at reading code than having to zoom into image to see things.
+And many of our users use screenreaders, which cannot read out images.
```
Comment (ee7) [inline community/good-member/writing-support-requests.md]:
```suggestion
And many of our users use screen readers, which cannot read out images.
```

I think.

**Review by ee7 — APPROVED** [review]

**Comment by iHiD** [comment]
Thanks, @ee7 :)

**Pull request merged**